### DISCUSSION TITLE
[Feature request] getMessagingActionsForOrder returns more than just a action links

### DISCUSSION THREAD

**Discussion by MichalOrm** [amzn/selling-partner-api-models#579] [Ideas]
At current state this resource is kinda pointless without some basic details such as:
- message id
- message date
- user name/id
- message body/content

Please consider adding all of above in near-future update

**Comment by nogamike76** [comment]
How do I do that

May God guide us and may we all, including myself find happiness, love and
forgiveness. Be kind to someone, help someone with something that would
help them. Make sure you tel your family you love them. May the kings be
protected and may our King, Jesus Christ, always be the divine and glory,
in honor of our father, the almighty one, God.
Spirit Leader
Michael Noga jr.
Descendant of King David
Nogah, son of David
T8<3
$€£¥
